server/base/pom.xml

```diff
@@ -152,5 +148,40 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>hadoop2</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
```

Comment:
I don't think `activeByDefault` does what you think it does here. `activeByDefault` is only active, when no other profiles are activated. If I activate another profile, like `-Papache-release`, this one will get disabled.

Reply:
Ugh, I hate this. Yes, I forgot about this miserable maven-ism.